### PR TITLE
feat(components/molecule/photoUploader): optional divider

### PR DESCRIPTION
**Pull request by andresz1**
## molecule/photoUploader

This PR makes `divider` optional

### Types of changes

- [x] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] 🧾 Documentation
- [ ] 📷 Demo
- [ ] 🧪 Test
- [ ] 🧠 Refactor
- [ ] 💄 Styles

### Description, Motivation and Context
A layout given by the design team without divider

Screenshot 2022-05-23 at 09 31 55

### CODE REVIEW

**Comment by andresin87** [comment]
Are you sure this is gonna be retrocompatible? I dnt think so

File: components/molecule/photoUploader/src/InitialState/index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
+import cx from 'classname'
```
Comment (andresin87) [inline components/molecule/photoUploader/src/InitialState/index.js]:
```suggestion
import cx from 'classnames'
```

**Comment by andresz1** [comment]
> Are you sure this is gonna be retrocompatible? I dnt think so

Yes I think so. Why not ?

**Comment by turolopezsanabria** [comment]
Add a demo, please

**Comment by andresz1** [comment]
> Add a demo, please

Not sure if would be worth it. This is just making the component work as expected if `dragPhotoDividerTextInitialContent` prop is set to `''` or `null`. What do you think ? cc @turolopezsanabria 

Screenshot 2022-05-23 at 09 58 06

**Comment by turolopezsanabria** [comment]
> > Add a demo, please
> 
> Not sure if would be worth it. This is just making the component work as expected if `dragPhotoDividerTextInitialContent` prop is set to `''` or `null`. What do you think ? cc @turolopezsanabria
> 
> Screenshot 2022-05-23 at 09 58 06

ah!!! Okidoki, I agree thnx

**Review by andresin87 — APPROVED** [review]

**Pull request merged**